gdi/get-data-in/application/otel-dotnet/instrumentation/connect-traces-logs.rst

```diff
@@ -0,0 +1,114 @@
+
+The following is an example of logs produced by a sample console application:
+
+.. code-block:: text
+
+   2023-09-15 15:23:17 2023-09-15T13:23:17.704Z    info    ResourceLog #0
+   2023-09-15 15:23:17 Resource SchemaURL: 
+   2023-09-15 15:23:17 Resource attributes:
+   2023-09-15 15:23:17      -> splunk.distro.version: Str(1.0.0.0)
+   2023-09-15 15:23:17      -> container.id: Str(c894cdb646a29616b5f713195cf810be898ca99c311cac8d9d25d8561dd6964b)
+   2023-09-15 15:23:17      -> telemetry.auto.version: Str(1.0.0)
+   2023-09-15 15:23:17      -> telemetry.sdk.name: Str(opentelemetry)
+   2023-09-15 15:23:17      -> telemetry.sdk.language: Str(dotnet)
+   2023-09-15 15:23:17      -> telemetry.sdk.version: Str(1.6.0)
+   2023-09-15 15:23:17      -> service.name: Str(Example.LogTraceCorrelation.Console)
+   2023-09-15 15:23:17      -> deployment.environment: Str(dev)
+   2023-09-15 15:23:17      -> service.version: Str(1.0.0)
+   2023-09-15 15:23:17 ScopeLogs #0
+   2023-09-15 15:23:17 ScopeLogs SchemaURL: 
+   2023-09-15 15:23:17 InstrumentationScope  
+   2023-09-15 15:23:17 LogRecord #0
+   2023-09-15 15:23:17 ObservedTimestamp: 2023-09-15 13:23:13.1358363 +0000 UTC
+   2023-09-15 15:23:17 Timestamp: 2023-09-15 13:23:13.1358363 +0000 UTC
+   2023-09-15 15:23:17 SeverityText: Information
+   2023-09-15 15:23:17 SeverityNumber: Info(9)
+   2023-09-15 15:23:17 Body: Str(Hello from {activity})
+   2023-09-15 15:23:17 Attributes:
+   2023-09-15 15:23:17      -> activity: Str(LogWrappingActivity)
+   2023-09-15 15:23:17 Trace ID: 17512c0247942df04fb30e6090eacb2c
+   2023-09-15 15:23:17 Span ID: dc281b062178e72f
+   2023-09-15 15:23:17 Flags: 1
+
+.. _dotnet-traces-logs-requirements-otel:
+
+Check compatibility and requirements
+====================================================
+
+.. note:: Automatic log to trace correlation only works for .NET applications. For .NET Framework use manual correlation. See :ref:`manual-trace-logs-correlation-otel`.
+
+``Microsoft.Extensions.Logging`` versions ``6.0.0`` and above are supported.
+
+.. _dotnet-otel-enable-log-correlation:
+
+Activate log correlation
+============================
+
+The instrumentation turns on automatic log to trace correlation by default.
+
+If you need to turn off logs enrichment and export, set any of the following environment variables to ``false``:
+
+- ``OTEL_DOTNET_AUTO_LOGS_ENABLED``
+- ``OTEL_DOTNET_AUTO_LOGS_INSTRUMENTATION_ENABLED``
+- ``OTEL_DOTNET_AUTO_LOGS_ILOGGER_INSTRUMENTATION_ENABLED``
+
+.. _dotnet-otel-include-trace-data:
+
+Trace metadata in log statements
+===================================================
+
+Individual log records, if exported inside scope of an active activity, will contain trace context information:
+
+* ``TraceId``
+* ``SpanId``
+* ``TraceFlags``
+
+:new-page:`OpenTelemetry resource <https://github.com/open-telemetry/opentelemetry-specification/tree/4f23dce407b6fcaba34a049df7c3d41cdd58cb77/specification/resource/semantic_conventions>`
```

Comment:
```suggestion
For more information on entities that can produce telemetry, like logs, see :new-page:`OpenTelemetry resource <https://github.com/open-telemetry/opentelemetry-specification/tree/4f23dce407b6fcaba34a049df7c3d41cdd58cb77/specification/resource/semantic_conventions>`.
```

Reply:
I prefer the current wording, but will let others express their opinion.

Reply:
alternative wording:

```md
Logs produced by an application always have the following set of OpenTelemetry [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/tree/4f23dce407b6fcaba34a049df7c3d41cdd58cb77/specification/resource/semantic_conventions):```

Reply:
@Kielek Could you make that a GH suggestion, so that @lachmatt or myself can apply it?

Reply:
changed in https://github.com/splunk/public-o11y-docs/pull/933/commits/6dda38615f5d105cc462675b50c52398dc6bc4a6